Sources/TCPConnection.swift

```diff
@@ -65,7 +70,7 @@ public final class TCPConnection: Connection {
         }
 
         if flush {
-            try self.flush()
+            try self.flush(timingOut: 200.milliseconds)
```

Comment:
This should actually stay just `flush()` (defined [here](https://github.com/open-swift/C7/blob/master/Sources/Stream.swift#L29)). The reason it didn't compile for you is because of the import :)